frontend/src/concepts/dashboard/DashboardModalFooter.tsx

```diff
@@ -32,7 +43,7 @@ const DashboardModalFooter: React.FC<DashboardModalFooterProps> = ({
     {error && (
       <StackItem>
         <Alert isInline variant="danger" title={alertTitle}>
-          {error.message}
+          {getErrorMessage(error)}
```

Comment:
Mmmmm great addition, I would even say that the `isAxiosErrorWithInnerMessage` could be a util method in other components, there are several places where we use axios, not sure if we are handling well this, we can further investigate and log something if that's the case.

Reply:
Yeah, just searching the repo for `error.message` reveals a number of places where this might be useful. I wonder what the best way is to follow up on that after this PR, do you think it warrants a Jira issue?